PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -268,7 +273,29 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
         }
         .store(in: &cancellables)
     }
-
 }
 
 #endif
+
+extension PacketTunnelProvider.SubscriptionConfiguration {
```

Comment:
I’m not sure this type makes sense as a subtype of the `PacketTunnelProvider`. It’s needed by the `NetworkProtectionDeviceManager` which is a dependency of the `PacketTunnelProvider` which creates a sort of circular dependency (not a serious one, but it is nonethless). Also the `PacketTunnelProvider` already quite bloated (we’ve got swiftlint warnings).